.github/CODEOWNERS

```diff
@@ -69,6 +69,9 @@
 
 /apps/services/search-indexer/                                      @island-is/stefna @island-is/kosmos-kaos @island-is/aranja
 /libs/content-search-toolkit/                                       @island-is/stefna @island-is/kosmos-kaos @island-is/aranja
+/libs/content-search-index-manager                                  @island-is/stefna @island-is/kosmos-kaos @island-is/aranja
```

Comment:
Trailing slash to indicate this is a folder?

Reply:
Fixed